.gitignore

```diff
@@ -1,5 +1,10 @@
 .DS_Store
-.idea
+.idea/*
+!.idea/IntelliLang.xml
```

Comment:
this is just so that the SQL in the migraiton file is syntax highlighted.

Reply:
just fyi it's possible to set a global git ignore on your machine so you don't need to add your IDE specific settings here

https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/